packages/integration-tests/projects/suite-web/tests/metadata/account-metadata.test.ts

```diff
@@ -5,12 +5,24 @@ import { rerouteMetadataToMockProvider, stubOpen } from '../../stubs/metadata';
 
 // fixture contains number of request that given provider needs go through this test scenario
 const fixtures = [
-    // todo: [27, 28] when taproot firmware (2.4.3) is in suite
-    { provider: 'dropbox', numberOfRequests: [25, 26] },
-    // todo: [10, 12] when taproot firmware (2.4.3) is in suite
-    { provider: 'google', numberOfRequests: [9, 12] },
+    {
+        provider: 'dropbox',
+        numberOfRequests: {
+            '2-master': [27, 28],
+            legacy: [25, 26],
+        },
+    },
+    {
+        provider: 'google',
+        numberOfRequests: {
+            '2-master': [9, 12],
+            legacy: [9, 12],
+        },
+    },
 ] as const;
 
+let startEmuParams;
```

Comment:
I'm not sure if it's ok to use variable in cypress test cases like this (maybe those `it` cases can run in parallel).

It's probably fine, but I don't see any other similar use of `let` variable in our tests.

I'm thinking about moving it [inside the forEach ](https://github.com/trezor/trezor-suite/pull/4570/files#diff-2a0756d899484fd15b467409daf6bb9a71b40c410d8821012053ebd62a82fa4dR34) or do not pass it from starEmu at all and add some function called from here and startEmu definition instead.

something like:
 ```javascript
 export const getEmulatorFirmwareVersion = process.env.FIRMWARE || releasesT2[0].version.join('.')
 ```